Microsoft.Identity.Web/StartupHelpers.cs

```diff
@@ -0,0 +1,145 @@
+using Microsoft.AspNetCore.Authentication;
```

Comment:
@irvinesunday  is this the same helper as the other repos you've migrated?

Reply:
@jthake-msft - This is the first ASP.NET Core sample project I'm migrating, and so it's the first time opportunity I'm getting to use the `Microsoft.Identity.Web` helper library along with this specific helper class `StartupHelper.cs` that helps in adding AAD v2.0 authentication along with MSAL support.

Reply:
@jthake-msft : Microsoft.Identity.Web (or equivalent) will become something at some point. Part of it might get to ASP.NET core directly. Part might become MSAL.Web. but that last thing won't happen in Q3.